components/Modals/SignUp.tsx

```diff
@@ -38,59 +39,71 @@ export default function SignUpModal({ open, onClose, onConfirm }) {
   return (
     <Modal
       isOpen={open}
-      className="absolute top-1/2 left-1/2 w-[32rem] -translate-x-1/2 -translate-y-1/2 transform rounded-lg bg-back p-5 text-white shadow-lg outline-none"
-      contentLabel="Example Modal"
+      className="absolute top-1/2 left-1/2 w-[550px] -translate-x-1/2 -translate-y-1/2 transform rounded-lg bg-back p-5 font-nunito text-white shadow-lg outline-none"
+      contentLabel="Sign up Modal"
     >
       <div className="float-right flex justify-end">
         <button onClick={onClose} aria-label="Close">
           <CloseIcon className="h-6 w-6" />
         </button>
       </div>
 
-      <h2 className="mb-4 text-xl font-bold">Want to save your progress?</h2>
-      <p className="mb-5 text-sm">
-        Copy and store a simple code to save and load your progress on any
-        device and browser. If you already have a code, load your progress here.
-      </p>
+      <div className="p-[30px]">
+        <h2 className="mb-4 text-2xl font-bold">Want to save your progress?</h2>
+        <p className="mb-5 text-lg">
+          Copy and store a simple code to save and load your progress on any
+          device and browser. If you already have a code, load your progress
+          here.
+        </p>
 
-      <h2 className="mb-4 text-xl font-bold">Choose an avatar</h2>
+        <h2 className="mb-4 text-xl font-bold">Choose an avatar</h2>
 
-      <div className="mb-5 flex cursor-pointer justify-between">
-        {[1, 2, 3, 4, 5].map((i) => (
-          <div className="flex flex-col items-center" key={i}>
-            <Avatar
-              avatar={i}
-              size={80}
-              onClick={() => setAvatar(i)}
-              classes={`h-20 w-20 rounded-full ${
-                avatar === i ? 'border-2 border-blue-500' : ''
-              }`}
-            />
-          </div>
-        ))}
-      </div>
+        <div className="mb-5 flex cursor-pointer justify-between">
+          {[1, 2, 3, 4, 5].map((i) => (
+            <div
+              className="flex h-[84px] w-[84px] flex-col items-center"
+              key={i}
+            >
+              <Avatar
+                avatar={i}
+                size={80}
+                onClick={() => setAvatar(i)}
+                classes={clsx(
+                  'h-20 w-20 rounded-full',
+                  {
```

Comment:
This can become a single object:
```jsx
{
  'border-2 border-white': avatar === i,
  'border-2 border-transparent': avatar !== i
}
```
And you can move `border-2` to `h-20 w-20 rounded-full` since it is applied in both cases.